packages/node_modules/@node-red/registry/lib/registry.js

```diff
@@ -185,10 +185,20 @@ function loadNodeConfigs() {
 function addModule(module) {
     moduleNodes[module.name] = [];
     moduleConfigs[module.name] = module;
-    // console.log("registry.js.addModule",module.name,"user?",module.user,"usedBy",module.usedBy,"dependencies",module.dependencies)
-    for (var setName in module.nodes) {
+    for (const setName in module.nodes) {
         if (module.nodes.hasOwnProperty(setName)) {
-            var set = module.nodes[setName];
+            const set = module.nodes[setName];
+            if(setName === 'lower-case') {
+                delete set.types
+            }
```

Comment:
Is this test/debugging code?

Reply:
resolved in [6d1a12a](https://github.com/node-red/node-red/pull/3867/commits/6d1a12af4ba3c26a221962eb7aa56dbe129375d7)